ui/StatusQ/src/StatusQ/Core/Utils/Utils.qml

```diff
@@ -279,4 +279,20 @@ QtObject {
     function stripHttpsAndwwwFromUrl(text) {
         return text.replace(/http(s)?(:)?(\/\/)?|(\/\/)?(www\.)?(\/)/gim, '')
     }
+
+    /**
+      - given a contiguous array of non repeating numbers from [0..totalCount-1]
+      - @return an array of @p n random numbers, sorted in ascending order
+      Example:
+        const arr = [0, 1, 2, 3, 4, 5]
+        const indexes = nSamples(3, 6) // pick 3 random numbers from an array of 6 elements [0..5]
+        console.log(indexes) -> Array[0, 4, 5] // example output
+      */
+    function nSamples(n, totalCount) {
```

Comment:
Thanks, will add the check

Otherwise, definitely not a prob here; in the current design, we verify just 4 words (might change but it's definitely gonna be sth small)